recipes/padel/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "padel" %}
+{% set version = "2.21" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.yapcwsoft.com/dd/padeldescriptor/versions/PaDEL-Descriptor%20v{{ version }}.zip
+  sha256: 9cfba80da884d67e79dc9777fbe842f84ee08b5457251eda37e5a2d8cfd80637
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  host:
+    - openjdk
+    - python >=3
+    - pip
+  run:
+    - openjdk
+    - python >=3
+    - pip
```

Comment:
Do you really need ``pip`` to run this project?

Reply:
You're right, pip is not required.